systemtest/010-inspect.bats

```diff
@@ -27,11 +27,19 @@ load helpers
     # Now run inspect locally
     run_skopeo inspect dir:$workdir
     inspect_local=$output
-
-    # Each SHA-named file must be listed in the output of 'inspect'
+    run_skopeo inspect --raw dir:$workdir
+    inspect_local_raw=$output
+    config_digest=$(jq -r '.config.digest' <<<"$inspect_local_raw")
+
+    # Each SHA-named layer file (but not the config) must be listed in the output of 'inspect'.
+    # As of Skopeo 1.6, (skopeo inspect)'s output lists layer digests,
```

Comment:
```suggestion
    # As of Skopeo 1.6, (skopeo inspect's) output lists layer digests,
```

Reply:
Although I think I'd just dro the parens all together.

Reply:
The parens are a somewhat-weird habit to quote shell commands in a way where copy-pasting both with and without the parens is a valid shell command (unlike using `"`, or `'`, let alone typographically-correct quotes); so moving the `'s` inside doesn’t work.

The parens are an habit from a specific niche period of the Fedora IRC universe, obscure enough to be confusing, I’ll drop them.